app/src/main/java/com/github/se/eventradar/ui/navigation/NavigationActions.kt

```diff
@@ -5,9 +5,10 @@ import androidx.navigation.NavGraph.Companion.findStartDestination
 import com.github.se.eventradar.R
 
 object Route {
-  const val LOGIN = "login/Login"
-  const val OVERVIEW = "overview/Overview"
-  const val MAP = "map/Map"
+    const val LOGIN = "login/Login"
+    const val OVERVIEW = "overview/Overview"
+    const val MAP = "map/Map"
+    const val SIGN_UP = "sign_up"
```

Comment:
Navigation branch has been push, you can repull main and add your `SIGN_UP` route

Reply:
Perfect! I have merged it and created my SIGNUP route. Note that I kept your semantic rules using SIGNUP rather than SIGN_UP.